notebooks/dev_summit_2021/Generate PDF Report/Generate PDF Report.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p>unnecessary comma b/w degrees and minutes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>